data_juicer/config/config.py

```diff
@@ -478,6 +490,37 @@ def display_config(cfg):
     print(table)
 
 
+def export_config(cfg, path, format='yaml', skip_none=True, skip_check=True,
+                  overwrite=False, multifile=True, branch=None):
+    """
+        save the config object, some params are from jsonargparse
+    :param cfg: cfg object to save (Namespace type)
+    :param path: the save path
+    :param format: 'yaml', 'json', 'json_indented', 'parser_mode'
+    :param skip_none: Whether to exclude entries whose value is None.
+    :param skip_check: Whether to skip parser checking.
+    :param overwrite: Whether to overwrite existing files.
+    :param multifile: Whether to save multiple config files
+        by using the __path__ metas.
+    :param branch:
```

Comment:
Need to add description to this arg. BTW, it seems that save() function doesn't have an argument of "branch", maybe recheck is needed.